dagmc_stats.py

```diff
@@ -3,15 +3,13 @@
 sys.path.append('/opt/tljh/user/lib/moab/lib/python3.6/site-packages/pymoab-5.1.0-py3.6-linux-x86_64.egg')
 from pymoab import core, types
 from pymoab.rng import Range
-
+import numpy as np
 def get_dagmc_tags(my_core):
     """
     Get a dictionary with the important tags for DAGMC geometries
-
     inputs
     ------
     my_core : a MOAB Core instance
-
     outputs
```

Comment:
you should add those blank lines back: PEP8 recommend blank line between section...